doc/ref_impl/cntt-ri/chapters/chapter04.md

```diff
@@ -92,3 +92,130 @@ Talk to the Lab Owners if special hardware or topology requirements are needed,
+
+Pharos lab infrastructure has the following objectives:
+
+* Provides secure, scalable, standard and HA environments for feature development
+* Supports the full Euphrates deployment lifecycle (this requires a bare-metal environment)
+* Supports functional and performance testing of the Euphrates release
+* Provides mechanisms and procedures for secure remote access to Pharos compliant environments for OPNFV community
+* Deploying OpenStack in a Virtualized environment is possible and will be useful, however it does not provide a fully
+  featured deployment and realistic test environment for the Euphrates release of OPNFV.
+
+The high level architecture is outlined in the following diagram:
+
+<img src="../figures/ch04_opnfv_pharos_high_level.jpg" title="OPNFV Pharos High Level Architecture">
+
+A pharos compliant OPNFV test-bed provides:
+
+* One CentOS/Ubuntu jump server (Foundation Node) which can be used to perform the OpenStack RI installation, or host any
+  additional software needed.  This server may also participate in the OpenStack cluster if desired instead of acting as
+  a dedicated services node.
+* 5 target nodes which can be used in any combination, such as:
+  * 3 controller nodes + 2 compute/storage nodes
+  * 1 controller node + 4 compute/storage nodes
+  * 1 controller node + 1 compute node + 3 storage nodes
+* A configured network with the ability to provide the following networks:
+  * Out-of-band Management: Used for access to the lights out (ILO/IMPI/Redfish) network for the purpose of managing the
+    bare metal aspects of the servers, such as power control, BIOS configuration, etc.
+  * External (DMZ): Used to provide VMs with Internet access.  Directly accessible from the VPN.
+  * Provisioning / In-band Management (Admin): to perform management operations on the hypervisor software for each
+    node.  Can also be used for bootstrapping images using PXE or other installation technologies.
+  * API Access (Public): Exposes all OpenStack APIs, including the OpenStack Networking API, to tenants.
+  * Tenant Transport (Private): Used for VM data communication within the cloud deployment. The IP addressing requirements
+    of this network depend on the OpenStack Networking plug-in in use and the network configuration choices of the virtual
+    networks made by the tenant.
+  * Storage Access (Storage): Exposes SDS services to client read/write requests. This is the data path for access to
+    the content of the storage nodes, and also doubles as the storage replication network to provide data replication
+    between storage nodes.
+  * OpenStack Management (Management): Used for internal communication between OpenStack Components.
+
+For the purpose of this RI, the following diagram illustrates the networks and node connectivity.
+
+<img src="../figures/ch04_opnfv_lab_topology.jpg" title="OPNFV Pharos Lab Topology">
+
+
+<a name="4.5.2"></a>
+## 4.5.2 Hardware Specification
+
+CPU:
```

Comment:
This section needs to be merged with 4.2 in this document (we need to make sure they are not conflicting each other)

Reply:
This is a copy of the current pharos specification and I'm not sure if we copy / paste it here, to point pointers to it, in the current 4.2 section.  The current 4.2 section is slightly "more" than a Pharos pod, in terms of storage and networking, based on what was originally proposed in the 4.2 section.